src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Animation/KeyTimeConverter.cs

```diff
@@ -195,7 +195,7 @@ public class KeyTimeConverter : TypeConverter
                                     keyTime.Percent * 100.0,
                                     destinationType);
 
-                            return returnValue + _percentCharacter[0].ToString();
+                            return string.Concat(returnValue, (ReadOnlySpan<char>)_percentCharacter);
```

Comment:
Would it be faster to change _percentCharacter to be a single char instead of an array of char ? I think the only reason that _percentCharacter was an array was because it's used in TrimEnd(System.Char) which was introduced in .Net 5.

Reply:
Not really faster, but it'd be a bit cleaner.